### DISCUSSION TITLE
Any ways to prevent chrome driver from seeing windows hello?

### DISCUSSION THREAD

**Discussion by BishoLv** [seleniumbase/SeleniumBase#3064] [General]
Hello, when i try to open some links it redirects me to another page if i have windows hello on my pc and if not, the link opens directly, so i want help to prevent chrome driver from seeing windows hello on my device?

**Comment by mdmintz** [comment]
I don't know anything about "Windows Hello". Maybe try Stack Overflow. Or extract the URLs from links and open them directly with the `sb.open(url)` method.
